frontend/src/v4/modules/risks/risks.sagas.ts

```diff
@@ -412,6 +411,16 @@ function* cloneRisk({ dialogId }) {
 		clonedProperties.descriptionThumbnail = `data:image/png;base64,${base64Image}`;
 	}
 
+	if (clonedProperties.viewpoint?.screenshot) {
+		const base64Image = yield imageUrlToBase64(activeRisk.descriptionThumbnail);
+		clonedProperties.viewpoint.screenshot = `data:image/png;base64,${base64Image}`;
+	}
+
+	if (clonedProperties.viewpoint?.screenshotSmall) {
```

Comment:
I think using the same method of getting the `screenshot` for `screenshotSmall` is causing the thumbnails to be stretched
![image](https://user-images.githubusercontent.com/62481250/211332151-8301b067-e490-4831-8680-c69a83b43789.png)